packages/odyssey-storybook/src/components/odyssey-mui/Button/Button.stories.tsx

```diff
@@ -62,34 +61,27 @@ const storybookMeta: Meta<ButtonProps> = {
 
 export default storybookMeta;
 
-const interactWithButton = async (
-  args: ButtonProps,
-  canvasElement: HTMLElement,
-  step: StepFunction<ReactRenderer, ButtonProps>,
-  action: string
-) => {
-  try {
+const interactWithButton =
+  ({ canvasElement, step }: PlaywrightProps<ButtonProps>) =>
+  async (args: ButtonProps, actionName: string, hoverState = false) => {
```

Comment:
Can we change this to use an object for the second function? I really don’t like passing unnamed booleans because they give no information to someone else looking at the tests.

```suggestion
const interactWithButton =
  ({ canvasElement, step }: PlaywrightProps<ButtonProps>) =>
  async ({ButtonProps, actionName, hoverState = false}: { args: ButtonProps, actionName: string, hoverState: boolean}) => {
```